css/index.css

```diff
@@ -103,44 +149,128 @@ p {
 
 .middle-content .boxes .box {
     width: 12.5%;
-    height: 100px;
-    background: black;
-    margin: 20px 2.5%;
+    height: 10rem;
+    margin: 3% 2.5%;
     color: white;
     display: flex;
     align-items: center;
     justify-content: center;
 }
 
```

Comment:
Great work using multiple class names to take advantage of getting all the styling from the parent and only overriding the color of each box.